instrumentation/common/src/main/java/com/splunk/opentelemetry/instrumentation/MetricsInstrumentationModule.java

```diff
@@ -32,7 +32,7 @@ protected MetricsInstrumentationModule(
 
   @Override
   public boolean defaultEnabled() {
-    boolean metricsRegistryPresent = !Metrics.globalRegistry.getRegistries().isEmpty();
-    return metricsRegistryPresent && super.defaultEnabled();
+    boolean metricsEnabled = Config.get().getBoolean("splunk.metrics.enabled", true);
```

Comment:
Metrics should be disabled by default
```suggestion
    boolean metricsEnabled = Config.get().getBoolean("splunk.metrics.enabled", false);
```